### ISSUE TITLE
@nx/devkit support (nx 16.0.0)

### ISSUE THREAD

**Issue by markfknight** [bojanbass/nx-aws#285]
The breaking change from `nx 16.0.0-beta.1`.

When migrations are run `@nrwl/devkit` is replaced by `@nx/devkit`

``` shell
> git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -

 >  NX   Affected criteria defaulted to --base=main --head=HEAD


 >  NX   Cannot find module '@nrwl/devkit'

   Require stack:
   - /../../node_modules/@nx-aws-plugin/nx-aws-cache/src/tasks-runner/runner.js
   - /../../node_modules/@nx-aws-plugin/nx-aws-cache/src/index.js
   - /../../node_modules/nx/src/tasks-runner/run-command.js
   - /../../node_modules/nx/src/command-line/affected.js
   - /../../node_modules/nx/src/command-line/nx-commands.js
   - /../../node_modules/nx/bin/init-local.js
   - /../../node_modules/nx/bin/nx.js
   Pass --verbose to see the stacktrace.

husky - pre-commit hook exited with code 1 (error)
```

Caused by [16069](https://github.com/nrwl/nx/pull/16069)

**Comment by ImanMahmoudinasab** [comment]
@bojanbass NX 16 is released on May 2. I think now this issue should get higher priority.
A similar migration: https://github.com/MansaGroup/nx-gcs-remote-cache/pull/318

**Comment by bojanbass** [comment]
I will prepare the change soon. Also I would be happy to accepts PRs for this change.

**Comment by bojanbass** [comment]
Released in v3.0.0.
